### PR TITLE
refs #8175 - certificates for dispatch router

### PR DESCRIPTION
**Pull request by stbenjam**
Using the existing broker ones just isn't feasible, it needs its own unfortunately...

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -93,6 +93,13 @@
 
   $messaging_client_cert = "${pki_dir}/qpid_client_striped.crt"
 
+  $qpid_dispatch_server_cert = "/etc/qpid-dispatch/server.crt"
+  $qpid_dispatch_client_cert = "/etc/qpid-dispatch/client.crt"
+  $qpid_dispatch_server_key  = "/etc/qpid-dispatch/server.key"
+  $qpid_dispatch_client_key  = "/etc/qpid-dispatch/client.key"
```
Comment (ehelms) [inline manifests/params.pp]: Are these the dispatch router defaults or expected locations? i.e. are we able to configure the dispatch router where to look for certs ?

File: manifests/qpid_router.pp [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  $deploy                 = $::certs::deploy,
+  $server_cert            = $::certs::qpid_dispatch_server_cert,
+  $client_cert            = $::certs::qpid_dispatch_client_cert,
+  $server_key             = $::certs::qpid_dispatch_server_key,
+  $client_key             = $::certs::qpid_dispatch_client_key,
+  $owner                  = $::certs::qpid_dispatch_owner,
+  $group                  = $::certs::qpid_dispatch_group,
+) inherits certs::params {
+
+  $server_keypair = "${hostname}-qdrouterd-server"
```
Comment (ehelms) [inline manifests/qpid_router.pp]: What is qdrouterd?

File: manifests/params.pp (hunk 1 above)
Comment (stbenjam) [inline manifests/params.pp]: No, where would you put them?

File: manifests/qpid_router.pp (hunk 2 above)
Comment (stbenjam) [inline manifests/qpid_router.pp]: **Q**pid **D**ispatch **Router** **D**aemon, but I renamed it to qpid_router to be consistent with the class name.

**Comment by stbenjam** [comment]
@ehelms Updated

**Comment by ehelms** [comment]
ACK

**Pull request merged**